JBPM-9979 JBPM-9807 release notes: mark async nodes at diagram & enri…

nmirasch opened the pull request:
…ch process instance logs

Adding release notes for https://issues.redhat.com/browse/JBPM-9979 and https://issues.redhat.com/browse/JBPM-9807

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/Release.7.67.0.Final-section.adoc:
```diff
@@ -0,0 +1,11 @@
+
+= New and Noteworthy in jBPM 7.67.0
+
+The following features were added to jBPM 7.67.0
```
```suggestion
The following features are added to jBPM 7.67.0
```

nmirasch replied:
@kaldesai in all other release sections is used 'were' instead 'are'. Does it make sense put this different?

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/active-async-node-instances-diagram.adoc:
```diff
@@ -0,0 +1,8 @@
+[id='async-mark-767']
+
+= Show highlight active asynchronous node instances at process instance diagrams
```
```suggestion
= Displays highlighted active asynchronous node instances at process instance diagrams
```

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/active-async-node-instances-diagram.adoc:
```diff
@@ -0,0 +1,8 @@
+Now, the asynchronous node instances are being highlighted changing their border color when they have been created but
+haven't been completed yet.
+When a node instance is highlighted, there are two options: The related job still is in progress or, the related job has
+failed. In both cases the related job can be tracked in the 'Jobs' section.
```
Just rephrased this section a little bit. Let me know if you have any questions. 

```suggestion
`The asynchronous node instances are now highlighted by changing their border color when they are created but have not been completed yet.
When an asynchronous node instance is highlighted, there are two options as follows:
* The related job is still in the progress
* The related job is failed
In both of the options, the related job is tracked in the *Jobs* section. `
```

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/conf-active-async-nodes-color-diagram.adoc:
```diff
@@ -0,0 +1,9 @@
+
+A new property `Process instance diagram active async node border color` was added in *Settings -> Process Administration* in {CENTRAL} to enable you to
+change the node colors used in process instance diagrams to highlight active async node instances.
+
```
```suggestion
In {CENTRAL}, a new *Process instance diagram active async node border color* property is now added in the *Settings* -> *Process Administration* section. This property enables you to change the node colors used in the process instance diagrams to highlight the active asynchronous node instances.
```

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/conf-active-async-nodes-color-diagram.adoc:
```diff
@@ -0,0 +1,9 @@
+A new property `Process instance diagram active async node border color` was added in *Settings -> Process Administration* in {CENTRAL} to enable you to
+change the node colors used in process instance diagrams to highlight active async node instances.
+
+When you modify these configurations, the change is applied to all process instance diagrams in {CENTRAL}.
```
```suggestion
When you modify these configurations, the change is applied to all the process instance diagrams in {CENTRAL}.
```

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/process-instance-logs-update.adoc:
```diff
@@ -0,0 +1,7 @@
+[[_jbpmreleasenotes7670_2]]
+
+= Process instance logs have been enriched
```
```suggestion
= Process instance logs are now improved
```

kaldesai commented on doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.67.0.Final/process-instance-logs-update.adoc:
```diff
@@ -0,0 +1,7 @@
+
+Now the process instance logs show more information about the *Delegated* and *Forwarded* tasks
+operations, showing in each case the user that is performing the operation and what is its target.
+
```
Just rephrased this section a little bit. Let me know if you have any questions.

```suggestion
The process instance logs now display more information about the *Delegated* and *Forwarded* tasks operations. In each case, these logs display which user is operating and what is the target of it.
```

barboras7 replied:
I think that this section doesn't clearly explain what is the benefit of the feature. I would rephrase to something like ".... which user is delegating/forwarding and who is the target user". WDYT, @kaldesai?

![Screenshot from 2022-03-21 08-31-42](https://user-images.githubusercontent.com/11525583/159220582-8404bef6-0332-423b-aee5-f3e7a754060e.png)

nmirasch replied:
@barboras7 sorry I didn't see this comment. It makes sense to me, let me add it, Thanks!

nmirasch replied:
@barboras7 updated

kaldesai reviewed (commented):
Hey @nmirasch, I have added a few suggestions. Otherwise, everything looks good to me. Let me know if you have any questions about the suggestions. Thank you :)

barboras7 commented:
@nmirasch I see no mention of what the default color for the async node is. I think it should be mentioned somewhere (so that the user can search in the docs, that if a node is red, it means that it's an async node in some state).

nmirasch commented:
@kaldesai, @barboras7 Thanks for the review.
I've updated the Release notes section with your suggestions @kaldesai 
@barboras7 I've added some changes in the process management section to try to add some information about the default colors meaning in the diagram. @kaldesai It will need your review. Just let me know is any change need to be done.

Thanks!

cristianonicolai commented:
@kaldesai can this be merged?

kaldesai commented:
> @kaldesai can this be merged?

@cristianonicolai Yes, we can merge this. I am merging now.
